### PR TITLE
RELATED: DP-1758 Adding an article about supported versions of GD.UI

### PR DESCRIPTION
**Pull request by evgeniabogranskaya**

### CODE REVIEW

**Comment by evgeniabogranskaya** [comment]
Pls do not merge - the content should go through a review process by PMs first.
@no23reason, can you please tag this PR with an appropriate tag? Thanks!

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/537/ : SUCCESS in 1m 05s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/317/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/538/ : SUCCESS in 44s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/318/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/539/ : SUCCESS in 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/319/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/540/ : SUCCESS in 43s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/320/ : SUCCESS in 6s

File: README.md [hunk 1]
```diff
@@ -19,21 +19,14 @@ This repository contains the source code of the [documentation](https://sdk.good
...
 
-- The latest major version will receive all new functionality and all bug fixes
-- The previous major version will only receive fixes to critical issues and security fixes. These fixes will be applied on top of last released version of the previous major
-
-The customers are encouraged to always use the latest version of the UI SDK.
-
-In case of using older versions, the user might face API incompatibility, performance or security issues.
-
-Please follow the installation instructions in [react-components repository](https://github.com/gooddata/gooddata-ui-sdk) to update to the newest version.
+Please follow the installation instructions in the [react-components repository](https://github.com/gooddata/gooddata-ui-sdk) to update to the latest version.
```
Comment (no23reason) [inline README.md:26]: It's no longer called react-components, but gooddata ui sdk

Comment (evgeniabogranskaya) [reply README.md]: Thanks - fixed!

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/541/ : SUCCESS in 1m 33s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/321/ : SUCCESS in 7s

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/542/ : SUCCESS in 58s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/63/ : SUCCESS in 13s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/156/ : SUCCESS in 14s

**Pull request merged**